src/main/java/hu/bearmaster/tutorial/matek/Számológép.java

```diff
@@ -34,6 +39,35 @@
         return felbontás;
     }
 
+    public int lnko(List<Integer> számok) {
+        if (számok.size() < 2) {
+            return -1;
+        }
+        List<PrímTényezősFelbontás> felbontások = new ArrayList<>();
+        for (int szám : számok) {
+            PrímTényezősFelbontás felbontás = prímFelbontás(szám);
+            felbontások.add(felbontás);
+        }
+        Set<Integer> közösPrímOsztók = new HashSet<>(felbontások.get(0).prímek());
+
+        for (PrímTényezősFelbontás felbontás : felbontások) {
+            közösPrímOsztók.retainAll(felbontás.prímek());
+        }
+
+        PrímTényezősFelbontás lnkoFelbontás = new PrímTényezősFelbontás();
+
+        for (int prím : közösPrímOsztók) {
+            int kitevő = Integer.MAX_VALUE;
+            for (PrímTényezősFelbontás felbontás : felbontások) {
+                int aktuálisKitevő = felbontás.getPrímTényező(prím).orElseThrow().getKitevő();
+                kitevő = aktuálisKitevő < kitevő ? aktuálisKitevő : kitevő;
```

Comment:
Mit szólsz ahhoz, hogy Math.min metódust használjuk itt?
```suggestion
                kitevő = Math.min(kitevő, aktuálisKitevő);
```